Appraisals

```diff
@@ -120,8 +120,6 @@ if Gem::Requirement.new('> 1.9.2').satisfied_by?(ruby_version)
     gem 'jbuilder', '~> 2.0'
     gem 'sdoc', '~> 0.4.0'
     gem 'bcrypt', '~> 3.1.7'
-    gem 'byebug'
```

Comment:
Oops I missed this, sorry... 1) I didn't even know you could do this and 2) I'm not sure how I feel about additional conditionals in Gemfiles. I'd rather just remove those dependencies because I don't think they're going to make an impact on how shoulda-matchers works. But I'll keep this in mind...